application/healthcare-services-data-manager/app.py

```diff
@@ -1,83 +1,52 @@
 from chalice import Chalice
+import hs_service
```

Comment:
Are we happy with the naming of this service ? Should we more explicit eg healthcareservices_service or hcs_service...